docs/Credits.md

```diff
@@ -42,7 +42,8 @@ Unless otherwise specified, all the following are from [the Noun Project](https:
 * [Bow](https://thenounproject.com/search/?q=bow&i=101736) By Arthur Shlain for Bowman
 * [Fishing Vessel](https://thenounproject.com/term/fishing-vessel/23815/) By Luis Prado for Work Boats
 * [Greek Trireme](https://thenounproject.com/search/?q=ancient%20boat&i=1626303) By Zachary McCune for Trireme
-* [Greek Trireme](https://thenounproject.com/search/?q=ancient%20boat&i=1626303) By Zachary McCune for Quinquereme
+* [Greek Trireme](https://thenounproject.com/search/?q=ancient%20boat&i=1626303) By Zachary McCune for Quinquereme // These credits are incorrect
```

Comment:
@SimonCeder  - do you remember? Nice image, but I can't find it on tnp.

Reply:
The attribution is correct, I modified it a bit which I believe is allowed under the license. African Forest Elephant and Carthage symbol are also modifications.

Reply:
You edited the second sail onto the Trireme... Nicely done. Should the credits mention that it's not a 'direct' usage?

Reply:
The Creative Commons license suggests the phrase "The original work has been modified." I'll make a PR labeling my modifications.